OrbitGl/TimeGraph.cpp

```diff
@@ -214,6 +215,26 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+//-----------------------------------------------------------------------------
+void TimeGraph::VerticalAlign(const TextBox* a_TextBox) {
+  if (!a_TextBox) return;
+  if (a_TextBox->GetPos() == Vec2::Zero())  // Have to update the information of
+                                            // this TextBox before align
+    UpdatePrimitives();
+  auto m_WorldTopLeftY = m_Canvas->GetWorldTopLeftY();
```

Comment:
We are trying to follow the Google C++ code style for new code, which means no prefixes to variables. So no `m_`s and `a_`s. It's fine to leave the old style when you're editing old code and want to minimize the change though.

Ref: https://google.github.io/styleguide/cppguide.html

Reply:
This also applies to the `//----`. No need to add them to new functions and feel free to remove them from exisiting functions when you're chaging them.

Reply:
Yes, I was following the previous name convention. 
So, could it be something like "world_top_left_y = ..."?
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Yes, exactly. Thanks